spec/unit/lib/temporal/testing_spec.rb

```diff
@@ -35,6 +35,21 @@ def execute; end
       Temporal::Testing.local! { example.run }
     end
 
+    describe 'Workflow.execute_locally' do
```

Comment:
I think this test belongs in `spec/unit/lib/temporal/workflow_spec.rb`

Reply:
While the fix is in code that's also used in production, the method I'm calling is in `lib/temporal/testing`.  LMK if you still want me to move it.
Also LMK if you think I should test a non-testing path.  I didn't bother because it doesn't feel like this really comes up in production since each workflow looked to have its own thread so there wouldn't be context corruption.